.travis.yml

```diff
@@ -60,6 +60,8 @@ before_script:
   - box server start name=$ENGINE debug=true
 
 script:
+  # Check formatting
+  - box run-script format:check
```

Comment:
This actually breaks the current file sets as not all of them adhere to it.  Can you remove this for now.

Reply:
That would miss the entire point which is to not let bad formatting get back in the code base.  If we don't have this check we will constantly be having formatting commits like this.

I've adjusted some of the formatting scripts, but the point is they _should_ all pass the `format:check`.